src/UnitOfWork.php

```diff
@@ -620,6 +645,138 @@ public function hydrateGraphId($oid, $gid)
     }
 
     /**
+     * Merges the state of the given detached entity into this UnitOfWork.
+     *
+     * @param object $entity
+     *
+     * @return object The managed copy of the entity.
+     */
+    public function merge($entity)
+    {
+        // TODO write me
+        trigger_error("Function not implemented.", E_USER_ERROR);
+    }
+
+    /**
+     * Detaches an entity from the persistence management. It's persistence will
+     * no longer be managed by Doctrine.
+     *
+     * @param object $entity The entity to detach.
+     *
+     * @return void
+     */
+    public function detach($entity)
+    {
+        $visited = array();
+
+        $this->doDetach($entity, $visited);
+    }
+
+    /**
+     * Executes a detach operation on the given entity.
+     *
+     * @param object  $entity
+     * @param array   $visited
+     * @param boolean $noCascade if true, don't cascade detach operation.
+     *
+     * @return void
+     */
+    private function doDetach($entity, array &$visited, $noCascade = false)
+    {
+        $oid = spl_object_hash($entity);
+
+        if (isset($visited[$oid])) {
+            return; // Prevent infinite recursion
+        }
+
+        $visited[$oid] = $entity; // mark visited
+
+        switch ($this->getEntityState($entity, self::STATE_DETACHED)) {
+            case self::STATE_MANAGED:
+                if ($this->isManaged($entity)) {
+                    $this->removeManaged($entity);
+                }
+
+                unset(
+                    $this->nodesScheduledForCreate[$oid],
+                    $this->nodesScheduledForUpdate[$oid],
+                    $this->nodesScheduledForDelete[$oid],
+                    $this->entityStates[$oid]
```

Comment:
No, BUT bear in mind that `RelationshipEntities` need the same functionality as nodes, in the future :)